pkgs/applications/video/obs-studio/v4l2sink.nix

```diff
@@ -1,42 +1,49 @@
-{ stdenv, fetchFromGitHub
-, cmake, pkgconfig, wrapQtAppsHook
-, obs-studio }:
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, qt5
+, obs-studio
+}:
 
-stdenv.mkDerivation {
-  pname = "obs-v4l2sink-unstable";
-  version = "20181012";
+stdenv.mkDerivation rec {
+  pname = "obs-v4l2sink";
+  version = "0.1.0";
 
   src = fetchFromGitHub {
     owner = "CatxFish";
     repo = "obs-v4l2sink";
-    rev = "1ec3c8ada0e1040d867ce567f177be55cd278378";
-    sha256 = "03ah91cm1qz26k90mfx51l0d598i9bcmw39lkikjs1msm4c9dfxx";
+    rev = version;
+    sha256 = "0l4lavaywih5lzwgxcbnvdrxhpvkrmh56li06s3aryikngxwsk3z";
   };
 
-  nativeBuildInputs = [ cmake pkgconfig wrapQtAppsHook ];
-  buildInputs = [ obs-studio ];
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [ qt5.qtbase.dev obs-studio ];
```

Comment:
I'm not sure whether `wrapQtAppsHook` is needed (or preferable) in this case, considering it's a library.
I tried to follow the obs-v4l2sink README's build steps as closely as possible and this approach seemed to work.

Reply:
cc @jtojnar

Reply:
`wrapQtAppsHook` would be a no-op if there are no executables, AFAIK.

Reply:
The original derivation builds just fine, so `wrapQtAppsHook` is at least supplying the qt5 libs (I've tested without `libsForQt5.callPackage` in `top-level/all-packages.nix`)

What I'm wondering is whether it would be better to drop the explicit `qt5.qtbase` dependency in favor of `wrapQtAppsHook`.

Reply:
All direct dependencies should be listed directly. Adding wrapper just to get some dependencies that maybe match the required dependencies sounds fragile.

Reply:
Perfect, the PR should be ready for merging then :)